types/office-js-preview/index.d.ts

```diff
@@ -17765,7 +17765,7 @@ declare namespace Excel {
         attributionUser: Excel.User;
         /**
          *
-         * Represents the task's due date, it is used for "Schedule" history record type.
+         * Represents the task's due date. It is used for "Schedule" history record type.
                     It is in UTC time zone. Can be set to `null` to remove the due date. It should be set together with `startDate` to avoid conflicts.
```

Comment:
Here and other places consecutive sentences begin "It is...". Should we merge them? e.g.
```suggestion
         * Represents the task's due date. It is used for "Schedule" history record type and is in UTC time zone. Can be set to `null` to remove the due date. It should be set together with `startDate` to avoid conflicts.
```

Reply:
It reads a little weird, but I like that they're separate sentences since they're different points of interest for the customer. It's "how it's used" and "how it's formatted".

Reply:
@alison-mk, if you have a strong opinion either way, please let me know and we'll get it updated in the source.

Reply:
What about something like: Represents the task's due date in UTC time zone. It is used for "Schedule" history record type.